### PR TITLE
feat: Adjusting to be compatible with isolated regions

### PR DESCRIPTION
**Pull request by RanbirAulakh**
*Issue #, if available:*

**Description of changes:**
- Adjusting some of the code changes so we can deploy this CDK into few regions

**Testing:**
- Ran Integ test and it succeeded for MR and TS. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/osml/model_runner/mr_dataplane.ts [hunk 1]
```diff
@@ -444,15 +444,72 @@ export class MRDataplane extends Construct {
       cluster: this.cluster,
       minHealthyPercent: 100,
       securityGroups: this.securityGroups,
-      vpcSubnets: props.osmlVpc.selectedSubnets
+      vpcSubnets: props.osmlVpc.selectedSubnets,
+      desiredCount: this.mrDataplaneConfig.MR_DEFAULT_DESIRE_COUNT
     });
 
+    interface LoggingOptions {
```
Comment (drduhe) [inline lib/osml/model_runner/mr_dataplane.ts:451]: I feel like this interface should be defined at the top of the file in the entire fluent bit/logging stuff should be migrated into it's own component... but we can spin up a task for that later

File: lib/osml/osml_vpc.ts [hunk 2]
```diff
@@ -110,6 +112,7 @@ export class OSMLVpc extends Construct {
       // Create a new VPC
       const vpc = new Vpc(this, "OSMLVPC", {
         vpcName: props.vpcName,
+        maxAzs: isIsoB ? 2 : 3,
```
Comment (drduhe) [inline lib/osml/osml_vpc.ts:115]: I feel like maxAzs should be moved into the configuration of the construct and then this should be set to 2 or 3 when it is created via the config.

**Review by drduhe — DISMISSED** [review]

File: lib/osml/model_runner/mr_dataplane.ts (hunk 1 above)
Comment (RanbirAulakh) [reply lib/osml/model_runner/mr_dataplane.ts]: Updated in Rev2. Please take a look.

File: lib/osml/osml_vpc.ts (hunk 2 above)
Comment (RanbirAulakh) [reply lib/osml/osml_vpc.ts:115]: I agreed. But its only pertaining to one region which only does 2. Majority of it have 3.

File: lib/osml/model_runner/monitoring/mr_logging_options.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        log_group_name: props.logGroup.logGroupName,
+        log_format: "json/emf",
+        log_key: "log",
+        log_stream_prefix: "${TASK_ID}/",
+        endpoint: `https://${logsEndpoint}`
+      }
+    };
+
+    this.logging = LogDrivers.firelens(
+      loggingOptions as FireLensLogDriverProps
```
Comment (drduhe) [inline lib/osml/model_runner/monitoring/mr_logging_options.ts:113]: Why don't we just get rid of the `LoggingOptions` interface and directly use the options in `FireLensLogDriverProps`?

**Review by drduhe — CHANGES_REQUESTED** [review]

File: lib/osml/model_runner/monitoring/mr_logging_options.ts (hunk 3 above)
Comment (RanbirAulakh) [reply lib/osml/model_runner/monitoring/mr_logging_options.ts]: Revised! Next revision is up!

**Review by drduhe — APPROVED** [review]
LGTM

**Pull request merged**